spec/event_tag_utils_spec.rb

```diff
@@ -38,10 +38,16 @@
     end
```

Comment:
Also add, 
1. Empty Array as argument
https://github.com/optimizely/python-sdk/blob/master/tests/helpers_tests/test_event_tag_utils.py#L35
2. Empty Hash 
https://github.com/optimizely/python-sdk/blob/master/tests/helpers_tests/test_event_tag_utils.py#L36

Reply:
Empty Array is already checked: https://github.com/msohailhussain/ruby-sdk/blob/4ecb13a7b04881d0e897a25db3ba6cbcb676d2e6/spec/event_tag_utils_spec.rb#L31

Only add relevant comments.